src/PowerTools/PowerTools.csproj

```diff
@@ -53,6 +53,8 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="EnvDTE" />
```

Comment:
I actually have this: https://github.com/ErikEJ/SqlCeToolbox/blob/master/src/GUI/SqlCe35Toolbox/VSToolbox.csproj#L83